Documentation/compatibility.md

```diff
@@ -1,31 +1,35 @@
 ---
-title: "Compatibility"
-description: "The Prometheus Operator supports a number of Kubernetes and Prometheus releases."
-lead: ""
-date: 2021-03-08T08:49:31+00:00
-draft: false
-images: []
-menu:
-  docs:
-    parent: "operator"
-weight: 200
+weight: 202
 toc: true
+title: Compatibility
+menu:
+    docs:
+        parent: operator
+lead: The Prometheus Operator supports a number of Kubernetes and Prometheus releases.
+images: []
+draft: false
+description: The Prometheus Operator supports a number of Kubernetes and Prometheus releases.
 ---
 
-The Prometheus Operator supports a number of Kubernetes and Prometheus releases.
+It is recommended to use versions of the components identical or close to the versions used by the operator's end-to-end test suite (the specific version numbers are listed below).
 
 ## Kubernetes
 
-The Prometheus Operator uses client-go to communicate with Kubernetes clusters. The supported Kubernetes cluster version is determined by client-go. The compatibility matrix for client-go and Kubernetes clusters can be found [here](https://github.com/kubernetes/client-go#compatibility-matrix). All additional compatibility is only best effort, or happens to still/already be supported. The currently used client-go version is "v4.0.0-beta.0".
+Due to the use of CustomResourceDefinitions, Kubernetes >= v1.7.0 is required.
```

Comment:
Are we 100% sure that 1.7.0 is the minimal required k8s version? Especially considering note that is removed in this PR:

> Due to the use of apiextensions.k8s.io/v1 CustomResourceDefinitions, prometheus-operator v0.39.0 onward requires Kubernetes >= v1.16.0.